docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server]]
+= {fleet-server}
+
+beta[]
+
+{fleet-server} is a component used to centrally manage {agent}s. It serves as a
+control plane for updating agent policies, collecting status information, and
+coordinating actions across agents.
+
+//QUESTION: I think it's worth breaking down in detail how data flows between
+// the agent, Fleet Server, Fleet, and the stack, but I might have some details
+// wrong here. Is this correct?
+
+{fleet-server} is the mechanism that connects {agent}s to {fleet} and handles
+all communication between an {agent}, {fleet}, {es}, and {kib}. {fleet-server}:
+
+* Receives status information from connected {agent}s
+* Retrieves agent policies from {fleet} and stores them in {es}
+* Ships agent policies to all connected {agent}s
+* Handles communication with {es} and {kib}
+
+image:images/fleet-server-communication.png[Fleet Server handles communication between {agent}, {fleet}, {es}, and {kib}]
+
+{fleet-server} runs as a separate service alongside {agent}. The agent uses a
+special policy that governs how {fleet-server} connects to other systems and
+manages agent requests. The agent typically runs as a dedicated {fleet-server},
+but you can optionally use it for data collection on self-managed clusters. For
+more details, refer to <<fleet-server-deployment>>.
+
+[discrete]
+[[fleet-server-compatibility]]
+== Compatibility
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later (our {ess-product}[hosted {ess}] on {ecloud} or
+a self-managed cluster)
+* {ece} 2.10 or later
```

Comment:
is this right?  I don't even know which ECE version will map to the coming 7.13 release... but anything already shipped of ES won't work with FS

Reply:
It was in one of the issues or specs. @ruflin What version do we need here?

Reply:
@zanbel I'm lost on exact ECE versions ^

@dedemorton For later, we should also not here the compatibility: See https://github.com/elastic/fleet-server#compatbility-and-upgrades

Reply:
I've added a note to document compatibility when we work on the upgrade docs: https://github.com/elastic/obs-dc-team/issues/504